Formula/sha1dc.rb

```diff
@@ -0,0 +1,25 @@
+class Sha1dc < Formula
+  desc "Tool to detect SHA-1 collisions in files, including SHAttered"
+  homepage "https://github.com/cr-marcstevens/sha1collisiondetection"
+  url "https://github.com/cr-marcstevens/sha1collisiondetection/archive/stable-v1.0.2.tar.gz"
+  sha256 "ab1a5624d12417b5af9ffe6662546b6932c39c0e4c4bef822851a2d0e53df790"
+
+  # The "master" branch is unusably broken and behind the
+  # "simplified_c90" branch that's the basis for release.
+  head "https://github.com/cr-marcstevens/sha1collisiondetection.git",
+       :using => :git
```

Comment:
`:using => :git` is assumed here, no need to write it explicitly, unless Homebrew has changed something that I missed.